trame_slicer/views/slice_view.py

```diff
@@ -256,3 +256,9 @@ def get_slab_type(
         self,
     ) -> Literal[vtk.VTK_IMAGE_SLAB_SUM, vtk.VTK_IMAGE_SLAB_MAX, vtk.VTK_IMAGE_SLAB_MIN, vtk.VTK_IMAGE_SLAB_MEAN]:
         return self.mrml_view_node.GetSlabReconstructionType()
+
+    def set_foreground_opacity(self, opacity: float):
+        self.logic.GetSliceCompositeNode().SetForegroundOpacity(opacity)
```

Comment:
shouldn't this return the return value of SetForegroundOpacity ?

Reply:
The [composite node setter](https://github.com/Slicer/Slicer/blob/main/Libs/MRML/Core/vtkMRMLSliceCompositeNode.cxx#L349) doesn't seem to return any value. Did you have something else in mind?

Reply:
My bad, I got confused, I thought that `vtkSetMacro()` was returning true in case of a new value, false in case of a no-op.
Not only `vtkSetMacro()` does not do that, but `vtkSetMacro()` is not even used here.
I was twice wrong. 😅